frontend/www/js/omegaup/components/course/Intro.test.ts

```diff
@@ -0,0 +1,63 @@
+import { mount } from '@vue/test-utils';
+
+import T from '../../lang';
+import type { types } from '../../api_types';
+
+import course_Intro from './Intro.vue';
+
+const courseDetails = {
+  admission_mode: 'registration',
+  alias: 'test-course',
+  archived: false,
+  assignments: [],
+  clarifications: [],
+  objective: 'Objetivo de prueba',
+  level: '',
+  needs_basic_information: true,
+  description: 'Test',
+  finish_time: new Date(),
+  is_curator: true,
+  is_admin: true,
+  name: 'Course name',
+  public: true,
+  requests_user_information: 'yes',
+  school_name: '',
+  show_scoreboard: false,
+  start_time: new Date(),
+  student_count: 1,
+  unlimited_duration: false,
+} as types.CourseDetails;
```

Comment:
```suggestion
const courseDetails: types.CourseDetails = {
  admission_mode: 'registration',
  alias: 'test-course',
  archived: false,
  assignments: [],
  clarifications: [],
  objective: 'Objetivo de prueba',
  level: '',
  needs_basic_information: true,
  description: 'Test',
  finish_time: new Date(),
  is_curator: true,
  is_admin: true,
  name: 'Course name',
  public: true,
  requests_user_information: 'yes',
  school_name: '',
  show_scoreboard: false,
  start_time: new Date(),
  student_count: 1,
  unlimited_duration: false,
};
```

cuando se está declarando una constante, siempre hay que ponerle el tipo al principio, así se evita el uso de `as `.